File: C_Programming/Practise_Programs/PP_099.c
/*
==============================================================================
 File Name   : PP_099.c

 Program     : Display a String

 Description : Initializes a character array with a string and displays
               the string using the standard output function.

 Author      : Shubham Gadhe

 Time Complexity  : O(n)
 Space Complexity : O(1)
==============================================================================
*/

#include<stdio.h>

int main()
{
    char str[] = "Jay Ganesh...";

    // Display the string.
    printf("%s\n", str);        

    return 0;
}

/*
==============================================================================
 Sample Output
------------------------------------------------------------------------------
Jay Ganesh...
==============================================================================
*/

File: Java_Programming/Practise_Programs/PP_099.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_099
//
//  Description       : This program displays all elements of
//                      an array using a for loop.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 12/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

import java.util.Scanner;

class PP_099
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int Arr[] = {10,20,30,40,50};

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      No input required.
//
//  Sample Output :
//
//      10
//      20
//      30
//      40
//      50
//
//////////////////////////////////////////////////////////////////